its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SolutionKindTest.java

```diff
@@ -19,18 +19,17 @@
  */
 package com.sonar.it.scanner.msbuild.sonarqube;
 
-import com.sonar.it.scanner.msbuild.utils.AnalysisContext;
-import com.sonar.it.scanner.msbuild.utils.AnalysisResult;
-import com.sonar.it.scanner.msbuild.utils.ContextExtension;
-import com.sonar.it.scanner.msbuild.utils.MSBuildMinVersion;
-import com.sonar.it.scanner.msbuild.utils.OSPlatform;
-import com.sonar.it.scanner.msbuild.utils.TestUtils;
+import com.sonar.it.scanner.msbuild.utils.*;
+
```

Comment:
That's what I was talking about with missing format config in IntelliJ.
```suggestion
import com.sonar.it.scanner.msbuild.utils.AnalysisContext;
import com.sonar.it.scanner.msbuild.utils.AnalysisResult;
import com.sonar.it.scanner.msbuild.utils.ContextExtension;
import com.sonar.it.scanner.msbuild.utils.MSBuildMinVersion;
import com.sonar.it.scanner.msbuild.utils.OSPlatform;
import com.sonar.it.scanner.msbuild.utils.TestUtils;
```

Reply:
I might have imported the wrong coding style config. thanks!

Reply:
A but no - it's fixed. I don't see this locally.